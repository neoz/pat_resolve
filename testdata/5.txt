;--------- (delete these lines to allow sigmake to read this file)
; add '+' at the start of a line to select a module
; add '-' if you are not sure about the selection
; do nothing if you want to exclude all modules

S_P_CoreLib_System_Buffers_RangeCharSearchValues_1_S_P_CoreLib_System_Buffers_SearchValues_FalseConst____ctor	09 CBD3 0FB7C28BD0450FB7C0458BD0442BD0450FB7CA66895112664489491066448941
S_P_CoreLib_System_Buffers_RangeCharSearchValues_1_S_P_CoreLib_System_Buffers_SearchValues_TrueConst____ctor	09 CBD3 0FB7C28BD0450FB7C0458BD0442BD0450FB7CA66895112664489491066448941

Object__Equals                                    	00 0000 33C0483BCA0F94C0C3..............................................
S_P_CoreLib_System_Reflection_Runtime_TypeInfos_NativeFormat_NativeFormatRuntimeNamedTypeInfo__Equals	00 0000 33C0483BCA0F94C0C3..............................................
S_P_CoreLib_System_Reflection_Runtime_TypeInfos_RuntimeNamedTypeInfo__Equals	00 0000 33C0483BCA0F94C0C3..............................................
S_P_CoreLib_System_Reflection_Runtime_TypeInfos_RuntimeTypeInfo__Equals	00 0000 33C0483BCA0F94C0C3..............................................
S_P_CoreLib_System_Reflection_Runtime_TypeInfos_RuntimeTypeInfo__Equals_0	00 0000 33C0483BCA0F94C0C3..............................................
