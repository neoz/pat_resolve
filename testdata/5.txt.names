S_P_CoreLib_System_Buffers_RangeCharSearchValues_1_S_P_CoreLib_System_Buffers_SearchValues_FalseConst____ctor