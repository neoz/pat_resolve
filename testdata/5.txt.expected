
+S_P_CoreLib_System_Buffers_RangeCharSearchValues_1_S_P_CoreLib_System_Buffers_SearchValues_FalseConst____ctor	09 CBD3 0FB7C28BD0450FB7C0458BD0442BD0450FB7CA66895112664489491066448941
S_P_CoreLib_System_Buffers_RangeCharSearchValues_1_S_P_CoreLib_System_Buffers_SearchValues_TrueConst____ctor	09 CBD3 0FB7C28BD0450FB7C0458BD0442BD0450FB7CA66895112664489491066448941

Object__Equals                                    	00 0000 33C0483BCA0F94C0C3..............................................
S_P_CoreLib_System_Reflection_Runtime_TypeInfos_NativeFormat_NativeFormatRuntimeNamedTypeInfo__Equals	00 0000 33C0483BCA0F94C0C3..............................................
S_P_CoreLib_System_Reflection_Runtime_TypeInfos_RuntimeNamedTypeInfo__Equals	00 0000 33C0483BCA0F94C0C3..............................................
S_P_CoreLib_System_Reflection_Runtime_TypeInfos_RuntimeTypeInfo__Equals	00 0000 33C0483BCA0F94C0C3..............................................
S_P_CoreLib_System_Reflection_Runtime_TypeInfos_RuntimeTypeInfo__Equals_0	00 0000 33C0483BCA0F94C0C3..............................................
